### PR TITLE
Notifiec

### PR DESCRIPTION
**Pull request by Cliftonz**
added discord notfication script

### CODE REVIEW

**Comment by Cliftonz** [comment]
Changes and updates are prepped and ready on Rundeck.

**Comment by TheOtherUnknown** [comment]
I'll poke though this on Rundeck sometime this weekend. Looks pretty good though

**Comment by Cliftonz** [comment]
@TheOtherUnknown thid is not put out on rundeck yet till it gets approved and merged to master, but the job is set up and ready to go. I would look at the phrasing or the notifications to make sure they sounds correct.

File: EventNotifier.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        if data[2] == 0 and (-epsilon < divmod(((now - event3d).total_seconds()), 60)[0] < epsilon):
+            # update db
+            cursor.execute(
+                f"UPDATE `csg_automations`.`eventNotification` SET `threeday` = 1 WHERE `eventId` = {eventid}")
+            conn.commit()
+            # send alert
+            webhook = DiscordWebhook(url=uri, content=f"@everyone Hey reminder bot here,\n CSG will be hosting "
+                                                      f"{event['summary']} in 3 days"
+                                                      f" on the {eventTime.day} at {eventTime.time}."
+                                                      f"We hope to see everyone their!\n"
```
Comment (TheOtherUnknown) [inline EventNotifier.py]: `
```suggestion
                                                      f"We hope to see everyone there!\n"
```

File: EventNotifier.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        if data[3] == 0 and (-epsilon < divmod(((now - deadline3d).total_seconds()), 60)[0] < epsilon):
+            # update db
+            cursor.execute(
+                f"UPDATE `csg_automations`.`eventNotification` SET `threeDay` = 1 WHERE `eventId` = {eventid};")
+            conn.commit()
+            # send alert
+            webhook = DiscordWebhook(url=uri,
+                                     content=f"@everyone Hey reminder bot here,\n The CSG has a deadline for"
+                                             f" {event['summary']} in 3 days. "
+                                             f"Please do not forget to sign up if your interested. \n"
```
Comment (TheOtherUnknown) [inline EventNotifier.py]:
```suggestion
                                             f"Please do not forget to sign up if you are interested. \n"
```

File: EventNotifier.py [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+        if data[4] == 0 and (-epsilon < divmod(((now - deadline1d).total_seconds()), 60)[0] < epsilon):
+            # update db
+            cursor.execute(
+                f"UPDATE `csg_automations`.`eventNotification` SET `oneDay` = 1 WHERE `eventId` = {eventid};")
+            conn.commit()
+            # send alert
+            webhook = DiscordWebhook(url=uri,
+                                     content=f"@everyone Hey reminder bot here,\n The CSG has a deadline for"
+                                             f" {event['summary']} in 1 days."
+                                             f" Please do not forget to sign up if your interested. \n"
```
Comment (TheOtherUnknown) [inline EventNotifier.py]:
```suggestion
                                             f" Please do not forget to sign up if you are interested. \n"
```

File: EventNotifier.py [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+        if data[4] == 0 and (-epsilon < divmod(((now - event1d).total_seconds()), 60)[0] < epsilon):
+            # update db
+            cursor.execute(
+                f"UPDATE `csg_automations`.`eventNotification` SET `oneDay` = 1 WHERE `eventId` = {eventid}")
+            conn.commit()
+            # send alert
+            webhook = DiscordWebhook(url=uri,
+                                     content=f"@everyone Hey reminder bot here,\n CSG will be hosting {event['summary']}"
+                                             f" tomorrow, the {eventTime.day} at {eventTime.time}. "
+                                             f"We hope to see everyone their!\n"
```
Comment (TheOtherUnknown) [inline EventNotifier.py]:
```suggestion
                                             f"We hope to see everyone there!\n"
```

**Review by TheOtherUnknown — CHANGES_REQUESTED** [review]
Few typos, but looks okay. I'll be interested to see this working.

**Review by AClifton22 — COMMENTED** [review]
I agree with Adam's changes.

**Review by AClifton22 — APPROVED** [review]
Approve changes

**Pull request merged**